assets/scss/components/_form.scss

```diff
@@ -13,6 +13,25 @@
   margin-bottom: govuk-spacing(7);
 }
 
+.hidden-conditional {
+  margin-bottom: -20px !important;
+
+  & > * {
+    display: none;
+  }
+}
+
+.govuk-checkboxes__item, .govuk-radios__item {
+  & [data-hidden] {
+    width: 0;
+    height: 0;
```

Comment:
How come you set the width/height to 0 - rather than `display:none`?

Reply:
Good catch! Originally I also wanted to apply negative top margin to the hidden element, hence why width/height=0. Turned out it wasn't needed

Reply:
resolved in https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/pull/811/commits/e59ce6f4c5cc938b368e621d03924290970c97f4